Formula/questdb.rb

```diff
@@ -20,46 +20,10 @@ def install
     (bin/"questdb").write_env_script libexec/"questdb.sh", Language::Java.overridable_java_home_env("11")
   end
 
-  plist_options manual: "questdb start"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-        <dict>
-          <key>KeepAlive</key>
-          <dict>
-            <key>SuccessfulExit</key>
-            <false/>
-          </dict>
-          <key>Label</key>
-          <string>#{plist_name}</string>
-          <key>ProgramArguments</key>
-          <array>
-            <string>#{opt_bin}/questdb</string>
-            <string>start</string>
-            <string>-d</string>
-            <string>var/"questdb"</string>
-            <string>-n</string>
-            <string>-f</string>
-          </array>
-          <key>RunAtLoad</key>
-          <true/>
-          <key>WorkingDirectory</key>
-          <string>#{var}/questdb</string>
-          <key>StandardErrorPath</key>
-          <string>#{var}/log/questdb.log</string>
-          <key>StandardOutPath</key>
-          <string>#{var}/log/questdb.log</string>
-          <key>SoftResourceLimits</key>
-          <dict>
-            <key>NumberOfFiles</key>
-            <integer>1024</integer>
-          </dict>
-        </dict>
-      </plist>
-    EOS
+  service do
+    run [opt_bin/"questdb", "start", "-d", "var/\"questdb\"", "-n", "-f"]
```

Comment:
```suggestion
    run [opt_bin/"questdb", "start", "-d", var/"questdb", "-n", "-f"]
```
I think the original plist had a typo. Also, this needs `log_path`.